lib/ansible/module_utils/memset.py

```diff
@@ -0,0 +1,134 @@
+# This code is part of Ansible, but is an independent component.
```

Comment:
@ryansb @s-hertel Do we want new modules directly in this directory, or do you want to move to `module_utils/cloud/memset/memset.py` as was done with network libraries in https://github.com/ansible/proposals/issues/76

Reply:
My rule of thumb is if there will be more than one module (memset_foo, memset_bar, ...) it's worth making a directory. So this would be a question for the contributor.